lib/livebook_web/live/session_live/app_teams_component.ex

```diff
@@ -7,6 +7,7 @@ defmodule LivebookWeb.SessionLive.AppTeamsComponent do
   def update(assigns, socket) do
     socket = assign(socket, assigns)
     deployment_groups = Provider.deployment_groups(assigns.hub)
+    app_deployments = Provider.app_deployments(assigns.hub)
```

Comment:
I wonder if these functions should be in `Livebook.Hubs.Team`, rather than on the protocol. Those are really specific to teams, the structs are all in the teams context too. @josevalim wdyt?

Reply:
should `AppDockerComponent` be accessible by Personal Hub?

Reply:
Ah, you are right. We need to keep Provider.deploument_groups, but we should remove Provider.app_deployments.